driver/mounter.go

```diff
@@ -363,3 +374,90 @@ func (m *mounter) IsMounted(target string) (bool, error) {
 
 	return targetFound, nil
 }
+
+// Copyright note for the functions below. Originally taken from
+// https://github.com/kubernetes/cloud-provider-openstack/blob/v1.16.0/pkg/volume/cinder/cinder_util.go
+// Sleightly modified.
+/*
+Copyright 2015 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+func (m *mounter) FindPath(logger *logrus.Entry, linuxSerial string) (*string, error) {
```

Comment:
This is just minor feedback: I'm not sure if the function's name is a good choice. 

I understand that the initial idea was to find the volume at either `/dev/disk/by-id/virtio-$linuxSerial` or `/dev/disk/by-id/scsi-$linuxSerial` and return whatever path was the right one.
Now, the function finishes the volume attachment process by rescanning the SCSI host and calling `udevadm`. 

I'm okay with `FindPath` but maybe something like `FinalizeVolumeAttachment` might make it clearer to understand what the function does.